bookmark/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+from users.utils import IsLoggedIn
+from rest_framework import status
+from rest_framework.response import Response
+from posts.models import Post
+from bookmark.models import Bookmark
+from rest_framework.views import APIView
+
+class CreateAndDeleteBookmark(APIView):
+    
+    def post(self,request):
+        try:
+            pk=request.data['pk']
+            post=Post.objects.get(pk=pk)
+            if post is None:
+                return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
This if condition won't do anything. Read [this](https://docs.djangoproject.com/en/3.0/topics/db/queries/#retrieving-a-single-object-with-get). Django doesn't return None if object is not found.

Reply:
Done